site/shovel-dynamic.xml

```diff
@@ -258,18 +330,55 @@ limitations under the License.
             shovel should never delete itself.
           </p>
           <p>
-            If set to <code>queue-length</code> then the shovel will
-            measure the length of the source queue when starting up,
-            and delete itself after it has transfered that many
-            messages.
-          </p>
-          <p>
             If set to an integer, then the shovel will transfer that
             number of messages before deleting itself.
           </p>
         </dd>
       </dl>
+      <h4>AMQP 1.0 Destination</h4>
+
+      <dl>
+        <dt><code>dest-uri</code></dt>
+        <dd>
+          <p>
+            The AMQP URI(s) for the
+            destination. Mandatory. See <code>src-uri</code> above. Mandatory.
```

Comment:
Pointing to `src-uri` above is confusing, as just above is a 1-line description in the AMQP 1.0 source. They both should refer to the long description in 0.9.1 source.